lib/algo/DMD.h

```diff
@@ -161,12 +182,27 @@ class DMD
     std::pair<Matrix*, Matrix*> phiMultEigs(double t);
 
     /**
-     * @brief Internal function to obtain the DMD modes.
+     * @brief Construct the DMD object.
      */
     void constructDMD(const Matrix* f_snapshots,
                       int rank,
                       int num_procs);
 
+    /**
+     * @brief Construct f_snapshots_minus and f_snapshots_plus
```

Comment:
It is not known here what `f_snapshots_minus` and `f_snapshots_plus` are. It would be clearer just to say it returns a pair of pointers to the minus and plus snapshot matrices, in that order. Also, based on the order, the name should be `computeMinusPlusSnapshotMatrices` (or the order should be reversed to match the current name).

Reply:
The routine is now renamed to `computeDMDSnapshotPair`, and the variables are renamed to `f_snapshots_in` and `f_snapshots_out` respectively,